examples/async_example.py

```diff
@@ -59,8 +61,8 @@
 
     if status == "TRANSCRIBED":
         # Getting a list of current jobs connected with your account
-        # The optional parameters limits limits the length of the list. Starting_after
-        # Cuts off the beginning x jobs off the list returned
+        # The optional parameters limits the length of the list.
+        # Starting_after cuts off the beginning x jobs off the list returned
```

Comment:
Not your fault but this xml doc is wrong. should be something like
`starting_after is a job id which causes the removal of all jobs from the list which were created before that job`